sunpy/data/_sample.py

```diff
@@ -19,7 +21,19 @@
 
 
 sampledata_dir = config.get("downloads", "sample_dir")
+_ntuple_diskusage = namedtuple('usage', 'total used free')
 
+def disk_usage(path):
+    """Return disk usage statistics about the given path.
+
+    Returned valus is a named tuple with attributes 'total', 'used' and
+    'free', which are the amount of total, used and free space, in bytes.
+    """
+    st = os.statvfs(path)
+    free = st.f_bavail * st.f_frsize
+    total = st.f_blocks * st.f_frsize
+    used = (st.f_blocks - st.f_bfree) * st.f_frsize
+    return _ntuple_diskusage(total, used, free)
```

Comment:
Here, free space is the count of the bytes that ordinary user is allowed to use i.e. it excludes reserved space.
hence , total -(used + free) = reserved free space